src/js/hooks.ts

```diff
@@ -50,3 +50,16 @@ export const useBeaconWithAuthIfAllowed = () => {
     }
   }, [dispatch, hasPermission]);
 };
+
+// ################### ROUTER HOOKS ###################
+
+export const getCurrentPage = (): string => {
```

Comment:
technically not a hook - doesn't use `use` prefix and window.location/location/pathname don't trigger re-render